client/platform/desktop/frontend/components/Settings.vue

```diff
@@ -235,11 +246,45 @@ export default defineComponent({
           Read only mode is on
         </v-alert>
 
+        <v-card-title>Synchronize Recents</v-card-title>
+        <v-card-subtitle v-if="settings">
+          Scan project directory (<b><u>{{ settings.dataPath }}</u></b>) to rediscover
+          datasets and update Recents page.  This is useful if you've manually deleted or moved
+          dataset folders around.  DIVE Desktop stores annotation files, metadata, and possibly
+          trancoded copies of your source media here.
+          <browser-link
+            display="inline"
+            href="https://kitware.github.io/dive/Dive-Desktop"
+          >
+            Check the docs for more info about the Project Data Storage Path.
+          </browser-link>
+        </v-card-subtitle>
+        <v-btn
+          text
+          outlined
+          class="mx-4"
+          @click="doAutodiscover"
+        >
+          <v-icon class="pr-2">
+            mdi-sync {{ autodiscoverLoading ? 'mdi-spin' : '' }}
+          </v-icon>
+          Sync recents with Project Data
+        </v-btn>
+
```

Comment:
Block clicking again while `autodisoverLoading` is active?

Reply:
As an aside to this, I thought there was no feedback available to the user before looking at the code because of how quickly it performed the scan.  I only noticed the actual rotation once I started copying folders and had like 3k+ folders in the directory.  Should there be some indication that it completed successfully and that you should go back to recents if it happens faster than the animation occurs with low number of folders or if the data folder is not on a network drive?